diff --git a/createpatch.sh b/createpatch.sh
index c0947f9..82521d1 100755
--- a/createpatch.sh
+++ b/createpatch.sh
@@ -1,4 +1,12 @@
 #!/bin/bash
+
+git clone --depth 1 \
+    --branch v6.12.51 \
+    https://git.kernel.org/pub/scm/linux/kernel/git/stable/linux.git \
+    /tmp/linux
+
+# exclude .git, *.patch, createpatch.sh
+# copy the remaining files to /tmp/linux/drivers/net/wireless/espressif
 find . \
     -path './.git' -prune -o \
     -type f \
